packages/esm-patient-list-management-app/src/lists-table/lists-table.component.tsx

```diff
@@ -268,17 +253,20 @@ const PatientListStarIcon: React.FC<PatientListStarIconProps> = ({ cohortUuid, i
   return (
     <TableCell className={`cds--table-column-menu ${styles.starButton}`} key={cohortUuid} style={{ cursor: 'pointer' }}>
       <Button
+        iconDescription="Star list"
```

Comment:
The tooltip above the Star list icon button was empty because of this `iconDescription` prop was missing.

Reply:
User-facing strings should always be run through the translation system.